packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/AppConfigurationImpl.kt

```diff
@@ -69,4 +69,24 @@ internal class AppConfigurationImpl(
         platformVersion = OS_VERSION,
         sdkVersion = io.realm.internal.SDK_VERSION
     ).freeze()
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
I guess we would need tests for these too. Maybe there are some from realm-java that we can grab.

Reply:
There aren't any equality tests. I've written a couple of basic ones instead.